vendor/brave-ios/Ads/BATBraveAds.mm

```diff
@@ -146,18 +149,38 @@ + (void)setEnvironment:(int)environment
 - (void)initializeIfAdsEnabled
 {
   if (![self isAdsServiceRunning] && self.enabled) {
+    self.databaseQueue = dispatch_queue_create("com.rewards.db-transactions", DISPATCH_QUEUE_SERIAL);
```

Comment:
This queue label already exists in BATBraveLedger and should be given one specifically for ads. If it's expected for ads & ledger to use the same queue for running DB transactions, then the queue will have to be pulled up to BATBraveRewards

Reply:
Fixed